core/ui/Actions/new-journal.tid

```diff
@@ -2,15 +2,14 @@ title: $:/core/ui/Actions/new-journal
 tags: $:/tags/Actions
 description: create a new journal tiddler
 
-\define get-tags() $(textFieldTags)$ $(tagsFieldTags)$
 \whitespace trim
-<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} textFieldTags={{$:/config/NewJournal/Tags}} tagsFieldTags={{$:/config/NewJournal/Tags!!tags}} journalText={{$:/config/NewJournal/Text}}>
-<$wikify name="journalTitle" text="<$macrocall $name='now' format=<<journalTitleTemplate>>/>">
+<$let journalTitleTemplate={{$:/config/NewJournal/Title}} textFieldTags={{$:/config/NewJournal/Tags}} tagsFieldTags={{$:/config/NewJournal/Tags!!tags}} journalText={{$:/config/NewJournal/Text}}>
+<$wikify name="journalTitle" text="<$transclude $variable='now' format=<<journalTitleTemplate>>/>">
 <$reveal type="nomatch" state=<<journalTitle>> text="">
-<$action-sendmessage $message="tm-new-tiddler" title=<<journalTitle>> tags=<<get-tags>> text={{{ [<journalTitle>get[]] }}}/>
+<$action-sendmessage $message="tm-new-tiddler" title=<<journalTitle>> tags=`$(textFieldTags)$ $(tagsFieldTags)$` text={{{ [<journalTitle>get[]] }}}/>
```

Comment:
Would it be possible to use a `<<f.get-tags>>` function definition instead of duplicating the text substitution.

Reply:
I've considered an implementation before, which uses the join operator to join them together (`[<textFieldTags>] [<tagsFieldTags>] +[join[]]`). Perhaps a function can be defined for that.